packages/eservice-template-process/src/services/eserviceTemplateService.ts

```diff
@@ -496,12 +481,6 @@ export function eserviceTemplateServiceBuilder(
         );
       }
 
-      const tenant = await retrieveTenant(
-        eserviceTemplate.data.creatorId,
-        readModelService
-      );
-      assertTenantKindExists(tenant);
```

Comment:
I understand we don't use the caller tenant kind for the risk analysis validation, but shouldn't we still check that the caller tenant exists and only remove the `assertTenantKindExists` assertion? 
If the creator tenant does not exist, something is wrong, and we should be ok throw (same in other methods of this file).
If you agree then you can also restore many changes in tests/errormappers/readmodelservice and reduce the diff of the PR

Reply:
Could you clarify why this tenant existence check is necessary? It seems redundant, especially when the caller is the tenant itself

Reply:
I think the back-end should always perform checks on the tenant existence: the caller could be calling via API with a token for the organization T and in the meantime the organization T could have been deleted for some reason.
It's an edge case, I don't even know if it's possible, but theoretically it is: if you don't check the tenant existence, and there is a valid token for tenant T, the token would be sufficient to perform the operation even if the tenant does not exist anymore.
If it's an impossible case, feel free to remove the check: no strong opinion...but in general I think the back-end should defend against all edge cases that are theoretically allowed by the data model